src/frontend/next/src/components/SnackBar/SnackBar.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, useState } from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+
+type SnackbarProps = {
+  message: string;
+};
+
+const SimpleSnackbar: FC<SnackbarProps> = ({ message }) => {
```

Comment:
I filed https://github.com/Seneca-CDOT/telescope/issues/1553 to move away from using `FC`, but you can leave it here if you like, and we can get rid of it later.  Up to you.

Reply:
@humphd : So if we move away from FC then should the approach be sth like below?
```ts
const SimpleSnackbar  = ({ message } : SnackbarProps) => {
```

Reply:
Exactly, just drop the FC type on the function.